nutils/evaluable.py

```diff
@@ -3685,6 +3685,12 @@ def divmod(x, y):
 def mod(arg1, arg2):
   return Mod(*_numpy_align(arg1, arg2))
 
+def greater(arg1, arg2):
+  return Greater(*_numpy_align(arg1, arg2))
```

Comment:
The reason for existence of `greater` and `less` seems to be `_numpy_align`. Isn't this something we'd like to get rid of?

Reply:
removed.